### ISSUE TITLE
I have a question about process for classification.

### ISSUE THREAD

**Issue by star4s** [ultralytics/yolov5#10435]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Hello.
I have a question about process for classification.

The training images for classification are small pixel size (50 X50), and predict images are a process that is possible in large images (1600 X 1200) with various backgrounds?

In the yolov5/classify/tutorial.ipynb , 
there are suit (0.38) and bow tie(0.19)  on the result. In the case, images for training  the suit and the bow tie are small size (35 X 35) and the predicted image size is 1280 X 720, isn't it?

**Comment by glenn-jocher** [comment]
@star4s training and inference should be run at similar --imgsz for best results

**Comment by star4s** [comment]
@glenn-jocher  So, In your case, How did you set train data size and inference data set?
I know your inference "ziddan.jpg" mage size was 1280 X 720. How about your train data size?

**Comment by glenn-jocher** [comment]
@star4s oh, for classification on ImageNet we used the default --imgsz 224. For smaller images like MNIST or CIFAR we use --imgsz 128

For custom data you could start at --imgsz 224 and increase from there if you need better results.

**Comment by star4s** [comment]
@glenn-jocher Could you share the date for classification training? I still don't understand for classification. I would like to inference  "ziddan.jpg" by the  classification weight after I trained your classification training date. I would like to train for classification by myself. Could you share the image labels for classification training?

**Comment by glenn-jocher** [comment]
@star4s Classification Usage examples are shown in README:
https://github.com/ultralytics/yolov5#classification--new

```
python classify/predict.py --weights yolov5s-cls.pt --data data/images/zidane.jpg
```

**Comment by star4s** [comment]
@glenn-jocher Thank you for your quick answer. I have three questions. 

1. How can I change model layer like your yolo Object Detection interface such as models/yolov5x6.yaml.

2. In addition, how can I change hyper-parameter like your yolo Object Detection interface such as data/hyps/hyp_evolve.yaml?
 
3. How can I save every weight per each epoch such as your yolo Object Detection command option like --save-period = 1 ?

**Comment by glenn-jocher** [comment]
@star4s you can modify these yamls directly or create new ones and then point them, i.e.
```
python train.py --cfg MODEL.yaml --hyps HYPS.yaml
```

**Comment by star4s** [comment]
@glenn-jocher Thank you for your quick answer. I have three questions.
your mention was "python train.py --cfg MODEL.yaml --hyps HYPS.yaml"
But I ask about classification. I did not ask on Object Detection.
Could you give me information about the 3 questions? 
Thank you for your attention.

**Comment by glenn-jocher** [comment]
@star4s Apologies for the mix-up. For classification, you can modify the model architecture and hyperparameters in the classification.yaml and hyps.yaml files, respectively. Additionally, you can use the --save_period argument to save weights every epoch during training.
